src/scripts.js

```diff
@@ -82,6 +92,11 @@ export {
   clickedRecipe,
   getRecipeCard,
   ingredientsList,
-  allRecipes,
-  searchBar
+  landingPage,
+  ourViewBtn,
+  yourViewBtn,
+  allUserRecipes,
+  chooseView,
+  searchBar,
+  //allRecipes
```

Comment:
Has all recipes been replaced by landing page? Should we remove references to all recipes?

Reply:
Since we won't be using allUserRecipes, I can change this back to allRecipes to keep all our codes cohesive.

Reply:
done